### PR TITLE
DOCS-8322 Netsuite 9.3 Reference

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc [hunk 1]
```diff
@@ -0,0 +1,5554 @@
...
+| Show warnings as errors a| Boolean |  When this preference is enabled, warning messages generated by NetSuite are treated as errors that cause an exception, resulting in rejection of the request. |  false |
+| Disable mandatory custom field validation a| Boolean |  This preference affects the handling of custom fields that are configured in the UI to be mandatory. If you set this preference to true, the fields are not treated as required during SOAP web services requests. If the preference is set to false, the fields are required. |  false |
+| Disable system notes for custom fields a| Boolean |  System notes are system-generated records that track changes to a record, including changes to specific field values. This preference lets you disable the creation of system notes for changes to custom fields. Depending on your integration, using this preference may increase performance. |  false |
+| Ignore read-only fields a| Boolean |  This preference affects the way the system behaves when you mistakenly submit a value for a readonly field. If you set this preference to true, the system ignores these mistakes. |  false |
```
Comment (dhirsch1) [inline netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc]: read-only

File: netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc [hunk 2]
```diff
@@ -0,0 +1,5554 @@
...
+* NETSUITE:SOAP_FAULT
+* NETSUITE:TRIGGER
+* NETSUITE:UNKNOWN
+
+
+[[attachRecord]]
+=== Attach Record
+`<netsuite:attach-record>`
+
+Attaches a source or contact record - that is, the attachment - to another destination one Not all record types are supported as source, destination or contact.
```
Comment (dhirsch1) [inline netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc]: Missing a period after "one"

File: netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc [hunk 3]
```diff
@@ -0,0 +1,5554 @@
...
+* NETSUITE:UNKNOWN
+
+
+[[updateRecordsList]]
+=== Update Records List
+`<netsuite:update-records-list>`
+
+Used to update one or more existing records in the system by providing a list of records. The records to be updated are identified through the provided unique identifiers.
+
+This operation cannot be used with data mapping or DataSense, it was added to allow users to add a batch of records of any type in 1 call.
```
Comment (dhirsch1) [inline netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc]: Run-on sentence.

File: netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc [hunk 4]
```diff
@@ -0,0 +1,5554 @@
...
+| TLS configuration a| <<Tls>> |  Protocol to use for communication. Valid values are HTTP and HTTPS. Default value is HTTP. When using HTTPS the HTTP communication is secured using TLS / SSL. If HTTPS was configured as protocol then the user needs to configure at least the keystore in the tls:context child element of this listener-config. |  |
+| Host a| String |  The hostname of the HTTP proxy, for example localhost This field also requires the port to be set. |  |
+| Port a| Number |  The port number of the HTTP proxy, for example 3128. The port number must be specified if the hostname is also specified. |  |
+| Username a| String |  The username which should be supplied to the HTTP proxy on every request to NetSuite. This field is optional, since a user might want to pass through an unauthenticated HTTP proxy. |  |
+| Password a| String |  The password which would be supplied to the HTTP proxy on every request to NetSuite. This field is optional, since a user might want to pass through an unauthenticated HTTP proxy. |  |
+| Partner ID a| String |  The partnerId agreed with NetSuite upon enabling SSO |  | x
```
Comment (dhirsch1) [inline netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc]: Wording needs to be fixed

File: netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc [hunk 5]
```diff
@@ -0,0 +1,5554 @@
...
+| Username a| String |  The username which should be supplied to the HTTP proxy on every request to NetSuite. This field is optional, since a user might want to pass through an unauthenticated HTTP proxy. |  |
+| Password a| String |  The password which would be supplied to the HTTP proxy on every request to NetSuite. This field is optional, since a user might want to pass through an unauthenticated HTTP proxy. |  |
+| E-mail a| String |  The login email of both NetSuite UI and SuiteTalk |  | x
+| Password a| String |  The login password of both the NetSuite UI and SuiteTalk |  | x
+| Account a| String |  NetSuite SuiteTalk WebService account ID |  | x
+| Role ID a| String |  The ID of the role used to login in SuiteTalk, which determines the Processor privileges |  | x
+| Application ID a| String |  The ID related to the Integration record to use. Integration records are set up on the NetSuite environment. |  | x
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  |
```
Comment (dhirsch1) [inline netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc]: Second sentence needs a period here and elsewhere.

File: netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc [hunk 6]
```diff
@@ -0,0 +1,5554 @@
...
+[[rest-config_restlet-login]]
+===== RESTlet Login
+
+RESTlet Login Authentication
+
+====== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Read Timeout a| Number | The amount of time that the client will wait for a response before it times out. 0 specifies that the client will wait indefinitely.+++ |  +++60000+++ | {nbsp}
```
Comment (dhirsch1) [inline netsuite/9.3/modules/ROOT/pages/netsuite-reference.adoc]: This appears in the HTML with a + at the end.

**Review by mcherem — DISMISSED** [review]
Peer review.

**Review by kevintroller — APPROVED** [review]

**Pull request merged**